docs/build/program/_index.md

```diff
@@ -46,7 +46,8 @@ source .venv/bin/activate
 pip install viam-sdk
 ```
 
-Windows is not supported. If you are using Windows, use the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) and install the Python SDK using the preceding instructions for Linux.
+Windows is not supported.
```

Comment:
should we put this in an alert?

Reply:
I'm inclined to say no, because we already say "run this on a mac or linux computer" higher up in the page, and because there were vague murmurings that it might just work anyway??